onprc_ehr/src/org/labkey/onprc_ehr/notification/TreatmentAlertsFastsNotification.java

```diff
@@ -0,0 +1,65 @@
+package org.labkey.onprc_ehr.notification;
+
+import org.labkey.api.data.Container;
+import org.labkey.api.module.Module;
+import org.labkey.api.security.User;
+
+import java.util.Date;
+
+/**
+ * Created by kollil on 11/1/2021
+ */
+
+public class TreatmentAlertsFastsNotification extends ColonyAlertsNotification
+{
+    public TreatmentAlertsFastsNotification(Module owner)
+    {
+        super(owner);
+    }
+
+    @Override
+    public String getName()
+    {
+        return "Fasts treatment Notification";
+    }
+
+    @Override
+    public String getEmailSubject(Container c)
+    {
+        return "Fasts treatment Alerts: " + getDateTimeFormat(c).format(new Date());
+    }
+
+    @Override
+    public String getCronString()
+    {
+        return "0 0 5 * * ?";
```

Comment:
Is the constring here is what you intended to schedule?  I think that will run every 5 seconds.

Reply:
I believe that's 5AM every day. I always have a hard time remembering cron's structure, so I pasted it here to interpret it

https://www.freeformatter.com/cron-expression-generator-quartz.html

Reply:
Raymond, I have the timings right, but haven't worked on the cron setup yet. I wanted to test the query and data first, so, deployed the code into git.